webapp/pages/login.vue

```diff
@@ -7,19 +7,26 @@
 <script>
 import LoginForm from '~/components/LoginForm/LoginForm.vue'
 import { VERSION } from '~/constants/terms-and-conditions-version.js'
+import { mapGetters } from 'vuex'
 
 export default {
   layout: 'no-header',
   components: {
     LoginForm,
   },
+  computed: {
+    ...mapGetters({
+      user: 'auth/user',
+    }),
+  },
   asyncData({ store, redirect }) {
     if (store.getters['auth/user'].termsAndConditionsAgreedVersion === VERSION) {
       redirect('/')
     }
   },
   methods: {
     handleSuccess() {
+      this.$i18n.set(this.user.locale || 'en')
       this.$router.replace(this.$route.query.path || '/')
     },
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T15:23:05Z" title="Thursday, October 31st 2019, 4:23:05 pm +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,19 +7,30 @@
 <script>
 import LoginForm from '~/components/LoginForm/LoginForm.vue'
 import { VERSION } from '~/constants/terms-and-conditions-version.js'
+import { mapGetters } from 'vuex'
 
 export default {
   layout: 'no-header',
   components: {
     LoginForm,
   },
+  computed: {
+    ...mapGetters({
+      user: 'auth/user',
+    }),
+  },
   asyncData({ store, redirect }) {
     if (store.getters['auth/user'].termsAndConditionsAgreedVersion === VERSION) {
       redirect('/')
     }
   },
   methods: {
     handleSuccess() {
+      const currentLocale = this.$i18n.locale()
+      const userSettingLocale = this.user.locale
+      if (currentLocale !== userSettingLocale) {
```


I think the `if` statement is quite unnecessary here, isn't it?